src/extension/inlineEdits/vscode-node/inlineCompletionProvider.ts

```diff
@@ -303,6 +301,69 @@ export class InlineCompletionProviderImpl implements InlineCompletionItemProvide
 		}
 	}
 
+	private createNextEditorEditComletionItem(requestingPosition: Position,
+		nextEdit: { document: TextDocument; range: Range; insertText: string }
+	): Omit<NesCompletionItem, 'telemetryBuilder' | 'info' | 'showInlineEditMenu' | 'action' | 'wasShown'> {
+		// Display the next edit in the current document, but with a command to open the next edit in the other document.
+		// & range of this completion item will be the same as the current documents cursor position.
+		const range = new Range(requestingPosition, requestingPosition);
+		const displayLocation: InlineCompletionDisplayLocation = { range, label: GoToNextEdit };
+
+		const commandArgs: TextDocumentShowOptions = {
+			preserveFocus: false,
+			selection: nextEdit.range
+		};
+		const command: Command = {
+			command: 'vscode.open',
+			title: GoToNextEdit,
+			arguments: [nextEdit.document.uri, commandArgs]
+		};
+		return {
+			range,
+			insertText: nextEdit.insertText,
+			showRange: range,
+			command,
+			isInlineEdit: true,
+			displayLocation,
+			isEditInAnotherDocument: true
+		};
+	}
+
+	private createComletionList(
+		doc: IVSCodeObservableDocument,
+		document: TextDocument,
+		position: Position,
+		range: Range,
+		result: NonNullable<(NextEditResult | DiagnosticsNextEditResult)['result']>,
```

Comment:
Preferred Omitting so that most of the common stuff is left out, else too many args and the like.
This function will build only whats required.